test/fixtures/render/nullable.json

```diff
@@ -0,0 +1,410 @@
+{
+  "_version": "4.0",
+  "metadata": [],
+  "name": "",
+  "description": "",
+  "element": "category",
+  "resourceGroups": [
+    {
+      "name": "",
+      "description": "",
+      "resources": [
+        {
+          "element": "resource",
+          "name": "",
+          "description": "",
+          "uriTemplate": "/",
+          "model": {},
+          "parameters": [],
+          "actions": [
+            {
+              "name": "",
+              "description": "",
+              "method": "GET",
+              "parameters": [],
+              "attributes": {
+                "relation": "",
+                "uriTemplate": ""
+              },
+              "content": [],
+              "examples": [
+                {
+                  "name": "",
+                  "description": "",
+                  "requests": [],
+                  "responses": [
+                    {
+                      "name": "200",
+                      "description": "",
+                      "headers": [
+                        {
+                          "name": "Content-Type",
+                          "value": "application/json"
+                        }
+                      ],
+                      "body": "{\n  \"s1\": \"abc\",\n  \"s2\": \"def\",\n  \"s3\": \"\",\n  \"s4\": null,\n  \"s5\": null,\n  \"s6\": null,\n  \"s7\": \"xyz\",\n  \"s8\": \"xyz\"\n}",
```

Comment:
If I run `nullable.apib` through drafter currently I get the following body/content:

```
"content": "\"{\n  \\"s1\\": \\"abc\\",\n  \\"s2\\": \\"def\\",\n  \\"s3\\": \\"\\",\n  \\"s4\\": null,\n  \\"s5\\": null,\n  \\"s6\\": null,\n  \\"s7\\": \\"xyz\\",\n
\\"s8\\": \\"xyz\\"\n}\","
```

That includes `"s3":""`. Is that not correct?